src/app/pages/data-protection/cloudsync/cloudsync-form/cloudsync-form.component.ts

```diff
@@ -865,6 +877,12 @@ export class CloudsyncFormComponent implements FormConfiguration {
               this.setDisabled('bucket_input', true, true);
             }
 
+            if (targetProvider && targetProvider.name === 'GOOGLE_CLOUD_STORAGE') {
```

Comment:
It'll better if the Provider name is used as an enum type, I think.

Reply:
Ok, there's no suitable enum, so I'll ping you once I'll replace all the providers from this dropdown with the enum

<img width="228" alt="image" src="https://user-images.githubusercontent.com/20611516/180740632-877a59b4-1fc7-4d48-b793-9a554aa8951d.png">